core/src/App.html

```diff
@@ -239,6 +241,18 @@
     };
   };
 
+  const checkIfUrlMatchesTheDomain = (viewUrl, domain) => {
+    return viewUrl && (viewUrl.startsWith(domain) || viewUrl.startsWith('/'));
```

Comment:
This is too optimisitc :-) In case of a relative path we need to check if the domain == current location domain.